src/main/java/wolox/training/models/Book.java

```diff
@@ -134,6 +142,7 @@ public Integer getPages() {
     }
 
     public void setPages(Integer pages) {
+        Preconditions.checkNotNull(pages);
```

Comment:
After checking if it is null, you can check if the content of the page field is greater than 0. Add menssage into checkNotNull

Reply:
I put message in constant and added validation greater than 0.